src/poliastro/twobody/orbit.py

```diff
@@ -271,7 +273,7 @@ def from_coords(cls, attractor, coord):
         pos = coord_in_irf.cartesian.xyz
         vel = coord_in_irf.cartesian.differentials["s"].d_xyz
 
-        return cls.from_vectors(attractor, pos, vel)
+        return cls.from_vectors(attractor, pos, vel, epoch=coord.obstime)
```

Comment:
Minor nitpick: we are not testing that `orbit.epoch == coords.obstime` :)

Reply:
yes. I missed that :(